### PR TITLE
CANS-1165- Change delete reason text

### PR DESCRIPTION
**Pull request by KShresthaCwds**
## JIRA
https://osi-cwds.atlassian.net/browse/CANS-1165
## Description
Changed Delete Reason.

## Tests

- [x] I used TDD to develop the feature
- [x] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: app/javascript/Application/components/Assessment/AssessmentDeleteModal.test.js [hunk 1]
```diff
@@ -90,17 +116,17 @@ describe('AssessmentDeleteModal', () => {
...
         expect(
           wrapper
             .find('.list__single-value')
             .last()
             .text()
-        ).toEqual('Referral / Case closed')
-        expect(wrapper.state().selectedReason.value).toBe('Referral / Case closed')
-        expect(wrapper.find(Comment).exists()).toBe(false)
+        ).toEqual('Other')
+        expect(wrapper.find(Comment).exists()).toBe(true)
```
Comment (AK56) [inline app/javascript/Application/components/Assessment/AssessmentDeleteModal.test.js]: We could also add a check for the comment text to be same

Comment (KShresthaCwds) [reply app/javascript/Application/components/Assessment/AssessmentDeleteModal.test.js]: Done

File: app/javascript/Application/components/Assessment/AssessmentDeleteModal.test.js [hunk 2]
```diff
@@ -65,21 +65,47 @@ describe('AssessmentDeleteModal', () => {
...
+        expect(wrapper.find(Comment).exists()).toBe(false)
       })
 
-      it('still keep comment displayed after user inputted reason then select Other option again', () => {
+      it('the second option of select should be #Duplicate#', () => {
+        const wrapper = mountWrapper(true, toggleModalSpy, assessmentHistoryCallbackSpy)
+        simulateSelect(wrapper, 2)
+        expect(
+          wrapper
+            .find('.list__single-value')
```
Comment (TylerOSI) [inline app/javascript/Application/components/Assessment/AssessmentDeleteModal.test.js]: this selector seems like its library dependent -- if we upgrade the library or change libraries that add this class name, this test will brake. Its best to find a way of selecting the element that is not dependent on dependency implementation.

File: app/javascript/Application/components/Assessment/AssessmentDeleteModal.test.js [hunk 3]
```diff
@@ -65,21 +65,47 @@ describe('AssessmentDeleteModal', () => {
...
+        expect(wrapper.state().selectedReason.value).toBe('Duplicate')
+        expect(wrapper.find(Comment).exists()).toBe(false)
+      })
+
+      it('the third option of select should be #Unable to locate child/family#', () => {
+        const wrapper = mountWrapper(true, toggleModalSpy, assessmentHistoryCallbackSpy)
+        simulateSelect(wrapper, 3)
+        expect(
+          wrapper
+            .find('.list__single-value')
```
Comment (TylerOSI) [inline app/javascript/Application/components/Assessment/AssessmentDeleteModal.test.js]: same comment as above.

File: app/javascript/Application/components/Assessment/AssessmentDeleteModal.test.js (hunk 2 above)
Comment (KShresthaCwds) [reply app/javascript/Application/components/Assessment/AssessmentDeleteModal.test.js]: These tests were written using `list__single-value`, So I was following the same pattern. I will leave as it is for now. We can refactor in the future.

**Pull request merged**